spec/factories/pictures.rb

```diff
@@ -0,0 +1,7 @@
+include ActionDispatch::TestProcess
+
+FactoryGirl.define do
+  factory :picture do
+    image { fixture_file_upload( File.join(Rails.root, 'spec', 'fixtures', 'images', 'ruby.png'), 'image/png') }
```

Comment:
Metrics/LineLength: Line is too long. [112/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)